lib/galaxy/datatypes/converters/interval_to_tabix_converter.py

```diff
@@ -27,12 +27,12 @@ def main():
     if options.preset:
         # Preset type.
         pysam.tabix_index(filename=index_fname, preset=options.preset, keep_original=True,
-                          index_filename=out_fname)
+                          index=out_fname, force=True)
     else:
         # For interval files; column indices are 0-based.
         pysam.tabix_index(filename=index_fname, seq_col=(options.chrom_col - 1),
                           start_col=(options.start_col - 1), end_col=(options.end_col - 1),
-                          keep_original=True, index_filename=out_fname)
+                          keep_original=True, index=out_fname, force=True)
```

Comment:
that's a little confusing in upstream, since `index_filename` is a valid argument to `VariantFile`.

Reply:
Yeah.  Added it as 'index' in https://github.com/pysam-developers/pysam/pull/537/files#diff-8d0b5835bcad7252fb985e226800ba62R864, though external indices are referred to as both 'index_filename' and 'filepath_index' in various other parts of pysam :/